style.css

```diff
@@ -100,6 +107,367 @@ ul li {
   padding: 0;
 }
 
+/* wrapper section end */
+
+/* recent works section start */
+
+.recent-works {
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  padding-left: 24px;
+  padding-right: 24px;
+}
+
+.works-title {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  padding-top: 80px;
+}
+
+.recent-works .works-title h2 {
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  color: #1c1a19;
+  text-align: center;
+  padding-bottom: 24px;
+}
+
+.solid-line {
+  border: 1px solid #bbb;
+  width: 100%;
+}
+
+/* recent works section end */
+
+/* all projects multi section start */
+
+.my-main-works {
+  display: grid;
+  grid-template-columns: repeat(1, 1fr);
+  padding-left: 24px;
+  padding-right: 24px;
+}
```

Comment:
- [ ] Kindly fix the alignment of the cards. Left and right gaps/margin/padding should be the same for all cards according to the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471&t=3CbGHNhPHWBiQT4V-0).
![Screenshot from 2023-01-22 00-13-31](https://user-images.githubusercontent.com/60083437/213881194-337c2cde-7eec-42b8-89de-3f627b38214a.png)